samples/javascript/videoshare/proto/pubsub.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
```

Comment:
is this definition fixed? can we reference the protocol from some package?

Reply:
`UpstreamMessage` and `DownstreamMessage` are fix